stingray/deadtime/fad.py

```diff
@@ -279,21 +244,22 @@ def calculate_FAD_correction(lc1, lc2, segment_size, norm="none", gti=None,
 
     results = Table()
 
-    print("n: " + str(n))
+    print("M: " + str(M))
```

Comment:
This is probably a left-over from some diagnostic run and should be deleted? Alternatively, possibly turn into loggin event?

Reply:
Ooops